util/cmake/ctd/mime.types

```diff
@@ -7,33 +7,18 @@ application/x-gff gff
 application/x-gtf gtf
 application/x-razers razers
 application/x-eland eland
-application/x-masai raw
-application/x-sam sam
+application/x-sam sam sam.bgzf sam.gz sam.bz2
 application/x-bam bam
 application/x-afg afg
 application/x-dist dist
 application/x-dot dot
 application/x-newick newick
 application/x-alftsv alf.tsv
-application/x-txt txt
-application/x-fasta fasta fa
-application/x-fastq fastq fq
-application/x-masic masic mas
-application/x-gsi gsi
-application/x-z-gsi gsi.gz
-application/x-razers razers
-application/x-eland eland
-application/x-masai raw
-application/x-sam sam
-application/x-bam bam
-application/x-afg afg
-application/x-dist dist
-application/x-dot dot
-application/x-newick newick
-application/x-alftsv alf.tsv
-application/x-txt txt
+application/x-txt txt csv tsv tab log
 application/x-bam-coverage-tsv bam_coverage_tsv
 application/x-fq-stats-tsv fq_stats_tsv
 application/x-rabema-report-tsv rabema_report_tsv
 application/x-index lf.drp lf.drs lf.drv lf.pst sa.ind sa.len sa.val txt.concat txt.limits txt.size
 application/x-yara rid.concat rid.limits
+application/x-vcf vcf bed bigbed bedgraph
```

Comment:
I don't want to be stringent about this.
However, in my humble opinion, bed/ bigbed / bedgraphs are not directly related to variant calling.
So.. Is this OK?